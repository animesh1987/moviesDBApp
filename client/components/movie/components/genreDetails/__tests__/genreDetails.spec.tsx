import * as React from 'react';
import { create } from 'react-test-renderer';
import { shallow } from 'enzyme';

import { MovieGenreDetails } from '../index';

describe('Movie Details Test', () => {
  const genres = ['Test', 'Test 2'];

  it('renders correctly', () => {
    const component = create(
      <MovieGenreDetails genres={genres}/>)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
