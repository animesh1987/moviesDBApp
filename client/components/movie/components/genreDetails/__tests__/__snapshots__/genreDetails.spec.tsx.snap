// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Details Test renders correctly 1`] = `
<div
  className="genre-details"
>
  Test Test 2
</div>
`;
